src/components/toolbar/paginering/paginering.tsx

```diff
@@ -81,23 +85,12 @@ function Paginering(props: PagineringProps) {
                 disabled={erPaSisteSide || seAlle}
                 onClick={() => totalPaginering(side + 1, seAlle)}
                 data-testid="paginering_hoyre"
+                ariaLabel="Neste side"
             >
                 <HoyreChevron />
             </KnappPanel>
         </div>
     );
 }
 
-const mapStateToProps = (state): StateProps => {
-    return {
-        side: selectSide(state),
-        sideStorrelse: selectSideStorrelse(state),
-        seAlle: selectSeAlle(state)
-    };
-};
-
-const mapDispatchToProps = (dispatch, props: OwnProps) => ({
-    endrePaginering: (side, seAlle) => dispatch(pagineringSetup({side, seAlle}))
-});
-
-export default connect(mapStateToProps, mapDispatchToProps)(Paginering);
```

Comment:
Good that you refactored this, it was difficult to debug and understand logic with the previous logic